jekyll/_cci2/upgrade-2.14.1-to-2.15.0.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: classic-docs
+title: "Upgrading from CircleCI Enterprise Installation 2.14.01 to 2.15.0"
+category: [administration]
+order: 12
+description: "How to upgrade from CircleCI Enterprise Installation 2.14.01 to 2.15.0 when using externalized MongoDB"
+---
+
+This document provides instructions for System Administrators upgrading an existing CircleCI Enterprise Installation 2.14.01 to 2.15.0 in the following sections:
```

Comment:
This document provides links to instructions for upgrading MongoDB to v3.6. If you are running an external MongoDB with your CircleCI installation, this MOngoDB upgrade is required for use with CircleCI v2.15.0.